### PR TITLE
Add a global variable to check if long tests should be skipped

### PR DESCRIPTION
**Pull request by ahoppen**
This centralizes the check for the environment variable.

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

File: Sources/_SwiftSyntaxTestSupport/LongTestsDisabled.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+public var longTestsDisabled: Bool {
+  if let value = ProcessInfo.processInfo.environment["SKIP_LONG_TESTS"] {
```
Comment (bnbarham) [inline Sources/_SwiftSyntaxTestSupport/LongTestsDisabled.swift:16]: It'd be nice to namespace this one as well IMO

**Review by bnbarham — APPROVED** [review]
Much nicer, thanks. Could uppercase `value`/also check for `true` but doesn't really matter.

**Pull request merged**